src/azure-cli/azure/cli/command_modules/role/tests/latest/test_role.py

```diff
@@ -195,6 +195,18 @@ def test_create_for_rbac_argument_error(self):
         with self.assertRaisesRegex(ArgumentUsageError, 'both'):
             self.cmd('ad sp create-for-rbac --role {role}')
 
+    def test_create_for_rbac_service_management_reference(self):
+        self.kwargs.update({
+            'display_name': self.create_random_name('azure-cli-test-', 30),
+            'service_management_reference': '96524024-75b0-497b-ab38-0381399a6a9d'
```

Comment:
Just curious, if it's a dummy GUID, how did you make the test pass?

Reply:
I am using our testing tenant to run the live test. The testing tenant has no restriction on the GUID. Only in Microsoft tenant, a real Service Tree ID is needed.